packages/onboard-react/src/components/HardwareWalletSteps/AddressListTable.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, ReactNode, useCallback, useMemo, useState } from "react";
+import {
+  Button,
+  AddressTablePagination,
+  TableBase,
+  ColumnOptions,
+  TransactionId,
+  Align,
+} from "@sovryn/ui";
+import { Account } from "@sovryn/onboard-hw-common";
+import { utils } from "ethers";
+import styles from "./AddressListTable.module.css";
+
+const PER_PAGE = 5;
+
+type AddressListTableProps = {
+  items: Account[];
+  onAccountSelected: (account: Account) => void;
+};
+
+type Item = {
+  index: string;
+  address: ReactNode;
+  balance: string;
+  asset: string;
+  account: Account;
+};
+
+export const AddressListTable: FC<AddressListTableProps> = ({
+  items,
+  onAccountSelected,
+}) => {
+  const [selected, setSelected] = useState<Account>();
+  const [offset, setOffset] = useState(0);
+
+  const handleSelect = useCallback(
+    (item: Item) => setSelected(item.account),
+    [setSelected]
+  );
+
+  const handleConfirm = useCallback(
+    () => onAccountSelected(selected!),
+    [selected, onAccountSelected]
+  );
+
+  const paginatedItems: Item[] = useMemo(
+    () =>
+      items.slice(offset, offset + PER_PAGE).map((item, index) => ({
+        index: String(offset + index).padStart(2, "0"),
+        address: (
+          <TransactionId
+            value={item.address}
+            href={`https://explorer.rsk.co/address/${item.address}`}
```

Comment:
This is wrong, if I select `ETH` derivation path, I should be able to see my account on `Etherscan` on not on `RSK Explorer`.

Reply:
Explorer should be resolved to the chain dapp uses not derivation path, in this case dapp will use RSK.
Later, it will be possible to request chain we want user to connect (for cross chain flows), then both explorer link and asset symbol will be dynamic

Reply:
Makes sense but this should be customizable, so we probably want something like `blockExplorerUrl` prop. Or do you want to adjust that only once we have cross chain flows?

Reply:
@tiltom we already have this prop in the config. I will change to use it now, even if it will fallback to the default one until connector is improved for cross-chain flows